resources/sushy-tools/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
+
+RUN apt update && apt install -y libvirt-dev && cd /root/ && \
+    git clone https://opendev.org/openstack/sushy-tools.git && \
+    cd /root/sushy-tools && \
+    pip3 install -r requirements.txt && python3 setup.py build && \
+    python3 setup.py install && pip3 install six requests libvirt-python
```

Comment:
Dependencies will be installed by pip, not need to do it explicitly.

Also, why not have a `CMD` here?

Reply:
six requests and libvirt-python are not in requirements.txt but are required to run sushy-emulator.
I'll add the CMD

Reply:
I'll propose a fix for six and requests.

Reply:
ok, I removed six and requests. we still need libvirt-python